resources/lsb_release

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+. /etc/os-release
+
+identity() {
+    printf "Distribution ID:\t $NAME\n"
+}
+
+description() {
+    printf "Description:\t $PRETTY_NAME\n"
+}
+
+release() {
+    printf "Release:\t $VERSION_ID\n"
+}
+
+supported_modules() {
+    echo LSB version:
```

Comment:
Ubuntus `lsb_release` doesn't print the this line at all (_not_ an empty list) in case of no LSB modules are installed (which is default).

Reply:
Ah, great. I'll omit. What does it do if you explicitly request -v? Nothing?

Reply:
Prints a message to stderr saying that no LSB modules are installed.

Reply:
Ack. I guess it's not really that important here.